icommons_ext_tools/requirements/demo.txt

```diff
@@ -9,6 +9,6 @@ newrelic==2.8.0.7
 gunicorn
 
 # Note: In the demo environment we use the /master branch
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@master#egg=django-icommons-common
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.9.5#egg=django-icommons-common
```

Comment:
Yes, part of  larger discussion. But a quick note: we are already pegging qa to certain versions, and demo is more stable than qa and close to prod - hence pegging to same as qa